test/tests/imageserver/test_image_service_system.py

```diff
@@ -0,0 +1,385 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+
+Author(s):
+Norton Luo
+This test validate the system level function of RackHD image-service. This test include VMware ESXi install and node
+rediscover.It use image server to store ESXi image and microkernel used for RackHD discovery.
+You need put an config file in the /config directory
+'''
+
+import os
+import sys
+import time
+import flogging
+import random
+import fit_common
+import urllib2
+import pexpect
+import unittest
+import test_api_utils
+from nose.plugins.attrib import attr
+logs = flogging.get_loggers()
+
+
+@attr(all=False, regression=False, smoke=False, imageservice=False)
+class test_image_service_system(fit_common.unittest.TestCase):
+    def _get_serverip(self):
+        args = fit_common.fitargs()['unhandled_arguments']
+        for arg in args:
+            if "imageserver" in arg:
+                serverip = arg.split("=")[1]
+                return serverip

```

Comment:
what if no ""imageserver"in all args? Suggest  return None  or things like that in this function and do some check when calling the function